src/index.html

```diff
@@ -11,6 +11,30 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form action="#" method="POST"
+      class="container"
+      data-qa="big">
+
+      <input type="search" class="input big_input"
```

Comment:
```suggestion
      <input type="search" class="input"
```
немає великого, є звичайний і менший від звичайного, звичайний буде мати набір стилів для усіх інпутів, а маленький просто перевизначить потрібні поля